src/debug_clang.cc

```diff
@@ -208,79 +206,71 @@ void Debug::Clang::start(const std::string &command, const boost::filesystem::pa
             Terminal::get().async_print(std::string(buffer, n), true);
         }
       }
-      event_mutex.unlock();
       std::this_thread::sleep_for(std::chrono::milliseconds(200));
```

Comment:
That will leave the mutex locked unnecessarily for 200 msec. Don't know how critical that is. Could be solved by a dedicated scope.